### PR TITLE
support include yaml

### PR DESCRIPTION
**Pull request by mmsqe**

### CODE REVIEW

File: pystarport/expansion.py [hunk 1]
```diff
@@ -42,29 +41,22 @@ def _expand(value, variables):
...
+    merged = {}
+    parent = Path(config_path).parent
+    for d in [dotenv_from_config, dotenv_from_param]:
+        if d not in (None, '', {}):
+            env_path = parent.joinpath(d)
+            if not env_path.is_file():
+                raise ValueError(
+                    f"Dotenv specified in config but not found at path: {env_path}"
+                )
+            merged = {
```
Comment (yihuang) [inline pystarport/expansion.py:56]: I think this approach is not intuitive from user's point of view.

Comment (mmsqe) [reply pystarport/expansion.py]: just think dotenv merge is easier than yaml, but might need wait [merge](https://github.com/mmsqe/mergedeep/commit/8be9c7b5bc9403ac0a86cc281a7644f49363ce38) change get merged

File: pyproject.toml [hunk 2]
```diff
@@ -22,6 +22,8 @@ docker = "^4.3.1"
 bech32 = "^1.1.0"
 multitail2 = "^1.5.2"
 python-dotenv = "^0.19.2"
+pyyaml-include = "^1.3"
+mergedeep = {git = "https://github.com/mmsqe/mergedeep.git", rev = "nested-merge"}
```
Comment (yihuang) [inline pyproject.toml:26]: we already used a `jsonmerge`, what's the difference with this one?

File: pystarport/expansion.py [hunk 3]
```diff
@@ -43,13 +45,22 @@ def _expand(value, variables):
...
+    path = Path(config_path)
+    parent = path.parent
+    YamlIncludeConstructor.add_to_loader_class(loader_class=yaml.FullLoader, base_dir = parent)
+
+    with open(path) as f:
+        config = yaml.load(f, Loader=yaml.FullLoader)
+
+    include = config.pop("include", {})
+    if include:
+        config = merge(include, config, strategy=Strategy.ADDITIVE)
```
Comment (yihuang) [inline pystarport/expansion.py:57]: `ADDITIVE` sounds non-intuitive, I think people will expect `REPLACE`.

File: pyproject.toml (hunk 2 above)
Comment (mmsqe) [reply pyproject.toml:26]: just more strategies, but can use without nested merge

**Review by yihuang — COMMENTED** [review]
Can we can deprecate the dotenv feature now with this? seems not, right?

**Comment by mmsqe** [comment]
> Can we can deprecate the dotenv feature now with this? seems not, right?

no change in dotenv func, but why deprecate it, no harms to keep?

**Review by yihuang — APPROVED** [review]

**Pull request merged**